JustSaying/AwsTools/ErrorQueue.cs

```diff
@@ -38,7 +38,7 @@ public override async Task UpdateQueueAttributeAsync(SqsBasicConfiguration queue
 
             var request = new SetQueueAttributesRequest
             {
-                QueueUrl = Url,
+                QueueUrl = Url.ToString(),
```

Comment:
You _should_ be able to use the `AbsoluteUri` property, rather than `ToString()` it, given that these should always be HTTP(S) URIs, not `file://` and the like.